### PR TITLE
DEV -> MASTER Release PR for initial release

### PR DESCRIPTION
**Pull request by bozmik**
New community review for initial release.

### CODE REVIEW

**Comment by apeltzer** [comment]
Please do a final bump-versions on this one, and  then we should be good to go 👍🏻 @reganhayward @bozmik

**Review by apeltzer — APPROVED** [review]
Only release bump-version required - LGTM!

File: CHANGELOG.md
Comment (apeltzer) [inline CHANGELOG.md:6]: No codename? ;-) https://www.codenamegenerator.com/

**Review by apeltzer — APPROVED** [review]
Looks good to me :-)

**Comment by apeltzer** [comment]
@MaxUlysse is busy giving an nf-core/bytesize talk today, but someone from @nf-core/core might want to give this a shot :-)

**Comment by MaxUlysse** [comment]
yeah, sorry about that, I'll promise I'll have a look after this afternoon if it's not approved already

File: CHANGELOG.md
Comment (reganhayward) [reply CHANGELOG.md]: Good thinking - updated :-)

**Comment by reganhayward** [comment]
> yeah, sorry about that, I'll promise I'll have a look after this afternoon if it's not approved already

File: conf/test.config
Comment (MaxUlysse) [inline conf/test.config:16]: Why 4GB and not 6?

File: .github/workflows/awsfulltest.yml
Comment (MaxUlysse) [inline .github/workflows/awsfulltest.yml:43]: Why not using the test_full profile here?

**Comment by MaxUlysse** [comment]
Everything seems in order.
I apologize for the late review.

I really have only one question, why not using the `test_full` profile for the awsfulltest?

File: conf/test.config
Comment (bozmik) [reply conf/test.config:16]: Actually, there was no particular reason for that. I will set it to 6 back :)

Comment (reganhayward) [reply conf/test.config]: That was left over from testing on my virtual machine with only 4GB of memory :-)

File: .github/workflows/awsfulltest.yml
Comment (reganhayward) [reply .github/workflows/awsfulltest.yml]: An oversight - have updated!

**Review by MaxUlysse — APPROVED** [review]
LGTM

**Pull request merged**
